source/agora/utils/PrettyPrinter.d

```diff
@@ -331,12 +331,15 @@ private struct BlockHeaderFmt
     {
         try
         {
-            formattedWrite(sink, "Height: %d, Prev: %s, Root: %s, Enrollments: [%s]\nSignature: %s,\nValidators: %s",
+            formattedWrite(sink, "Height: %d, Prev: %s, Root: %s, Enrollments: [%s]\nSignature: %s,\nValidators: %s,\nRandomSeed: [%s],\nMPV: [%s]",
```

Comment:
Are we going to stick with MPV? I think `missing` or `slashed` might be better.

Reply:
@AndrejMitrovic gave the same opinion about that. I will change it to `Missing Validators`.

Reply:
Do you think `Missing` is better?

Reply:
In fact `Missing validators` is better.

Reply:
Slashed validators is better I think.

"Missing" could sound like they just haven't signed the block yet.

Reply:
Updated to 'Slashed validators`.